sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/__init__.py

```diff
@@ -2,18 +2,31 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-from .client import CertificateClient
-from .enums import ActionType, KeyCurveName, KeyType, SecretContentType, KeyUsageType
-from .models import AdministratorDetails, CertificatePolicy, Contact, LifetimeAction
+from ._client import CertificateClient
+from ._enums import ActionType, KeyCurveName, KeyType, SecretContentType, KeyUsageType
+from ._models import (
+    AdministratorDetails,
+    Certificate,
+    CertificatePolicy,
+    Contact,
+    DeletedCertificate,
+    LifetimeAction,
+    CertificateOperation,
+    CertificateProperties,
```

Comment:
Should we also include Issuer, IssuerProperties, and Error here?

Reply:
I think so. I've removed certificates changes from this PR though, we can sort those out separately.